cards/cards.css

```diff
@@ -0,0 +1,127 @@
+@import "../for-import/reset.css"; 
+@import "../for-import/colors.css";
+@import "../for-import/temp-buttons.css";
+@import "../for-import/typography.css";
+
+/* style for icon on top of image */
+.card-top {
+    position: relative;
+}
```

Comment:
one empty line can be added after each style block ends

Reply:
done